### PR TITLE
AZI-27 Update log forwarder automation to randomize EventHub namespace

### PR DESCRIPTION
**Pull request by iotmani**
### What does this PR do?

Add randomization to default Eventhub namespace as [it has a global scope](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/resource-name-rules#microsofteventhub).

`newGuid()` function [creates a globally unique id](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions-string#newguid).

### Motivation

https://datadoghq.atlassian.net/browse/AZI-27

### Testing Guidelines

Ran modified Powershell script via Azure cloud shell, and saw it creates resources with new namespace.

### Additional Notes

Documentation PRs will be tracked via https://github.com/DataDog/dogweb/pull/64042

### Types of changes

- [ ] Bug fix
- [x] New feature
- [ ] Breaking change
- [ ] Misc (docs, refactoring, dependency upgrade, etc.)

### Check all that apply

- [x] This PR's description is comprehensive
- [ ] This PR contains breaking changes that are documented in the description
- [ ] This PR introduces new APIs or parameters that are documented and unlikely to change in the foreseeable future
- [x] This PR impacts documentation, and it has been updated (or a ticket has been logged)
- [ ] This PR's changes are covered by the automated tests
- [ ] This PR collects user input/sensitive content into Datadog
- [x] This PR passes the integration tests (ask a Datadog member to run the tests)
- [x] This PR passes the unit tests 
- [x] This PR passes the installation tests (ask a Datadog member to run the tests)

### CODE REVIEW

**Comment by iotmani** [comment]
Removed Powershell function / its unit-tests in favor of default generation within the ARM template.

File: azure/eventhub_log_forwarder/README.md [hunk 1]
```diff
@@ -1,3 +1,25 @@
 # Eventhub Log Forwarder Deployment
 
-Coming Soon
+Powershell scripts to autoamtically create a 'log forwarding pipeline' using Azure EventHub to collect Azure Platform Logs.
```
Comment (claudiadadamo) [inline azure/eventhub_log_forwarder/README.md]: Typo - automatically

File: azure/eventhub_log_forwarder/README.md [hunk 2]
```diff
@@ -1,3 +1,25 @@
...
+
+At a high level, the scripts call Azure Powershell functions to create and deploy Azure resources, mostly defined via a URL to a json template.
+
+The JSON template format is a [Azure Resource Manager (ARM)](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/overview), which can store variables, define resources to create, dependencies... etc. And can be validated via e.g. a Visual Studio Code plugin mentioned therein (under Authoring tools). 
+
+The exact steps these scripts automate are found in the ['Manual installation' of Log collection documentation](https://docs.datadoghq.com/integrations/azure/?tab=manualinstallation#log-collection).
+
+# Contributions
```
Comment (claudiadadamo) [inline azure/eventhub_log_forwarder/README.md]: One thing to note is that this repo is public and we're not taking non-Datadog contributions, so maybe this should go into internal documentation instead?

**Review by claudiadadamo — COMMENTED** [review]
2 tiny comments

**Review by claudiadadamo — APPROVED** [review]
lgtm

**Comment by iotmani** [comment]
Merging now that https://github.com/DataDog/dogweb/pull/64042 is merged.

**Pull request merged**